server/Services/GameRound.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using Api.Hubs;
+using Api.Extensions;
+using System.Threading;
+using Microsoft.AspNetCore.SignalR;
+using Api.DTO;
+
+namespace Api.Services
+{
+  public class GameRound
+  {
+    private readonly int EggCards = 10;
+    private readonly int HenCards = 5;
+    private readonly int FoxCards = 5;
+
+    private Timer GameTimer;
+
+    private List<CardType> Deck { get; set; }
+    private List<CardType> Pile { get; set; }
+
+    private bool CorrectCount(int count) { return count == 5; }
```

Comment:
Shouldn't it be `>= 5`? If I hit while the count is 6, I am supposed to win.

Reply:
Ok, was not sure of that rule, I will update the wiki.